internal/signer/linux/pkcs11/pkcs11_test.go

```diff
@@ -34,3 +40,79 @@ func TestParseHexStringFailure(t *testing.T) {
 		t.Error("Expected error but got nil")
 	}
 }
+
+func TestEncryptRSA(t *testing.T) {
+	key, _ := Cred("/usr/local/lib/softhsm/libsofthsm2.so","0x268c8a20","Demo Object","0000")
```

Comment:
Make these test params globals and wrap the credential creation in a helper function.

We need a strategy for where the softhsm library should live. It's probably best to assume a debian `apt` install location for now, since we only needed the debug build for our own development.

Reply:
I created a helper function along with global variables. Are there some other steps I should take to address the second half of your comment?

Reply:
Use the original path from `go-pkcs11` that's the location that is installed to by apt.

I'm thinking we should just symlink to `$HOME/.local/lib/softhsm` and see if that works, but let's do that later.